specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/longTermRetention.json

```diff
@@ -0,0 +1,724 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2017-03-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Sql/locations/{locationName}/servers/{serverName}/databases/{databaseName}/longTermRetentionBackups/{backupName}": {
+      "get": {
+        "tags": [
+          "LongTermRetentionBackups"
+        ],
+        "description": "Gets a long term retention backup.",
+        "operationId": "LongTermRetentionBackups_Get",
```

Comment:
there is no `LongTermRetentionBackups_CreateOrUpdate` operation, just curious is it the case that service just creates backups at regular interval without explicitly asking for it.

Reply:
Yes, Azure SQL DB takes full backups (which is what we store as part of LTR) once a week.